bundles/org.openhab.binding.lutron/README.md

```diff
@@ -337,16 +379,16 @@ A setting of 0 will disabled polling.
 You can also initiate a poll at any time by sending a refresh command (RefreshType.REFRESH) to the step channel.
 Note that it should usually be unnecessary for the poll interval to be set to less than 5-10 minutes, since the green mode step typically changes rather infrequently and takes effect gradually.
 
-Example:
+Thing configuration file example:
 
 ```
 Thing greenmode greenmode [ integrationId=22 ]
 ```
 
 ### Timeclock
 
-Radio RA2 and Homeworks QS have timeclock subsystems that provide scheduled execution of tasks at set times, randomized times, or at arbitrary offsets from local sunrise/sunset.
-The tasks executed depend on which timeclock mode (e.g. Normal, Away, Suspend) is currently selected, and the modes themselves are user-definable (Radio RA2 only).
+RadioRA 2 and Homeworks QS have timeclock subsystems that provide scheduled execution of tasks at set times, randomized times, or at arbitrary offsets from local sunrise/sunset.
+The tasks executed depend on which timeclock mode (e.g. Normal, Away, Suspend) is currently selected, and the modes themselves are user-definable (RadioRA 2 only).
```

Comment:
```suggestion
The tasks executed depend on the currently selected timeclock mode (e.g. Normal, Away, Suspend) and the modes themselves are user-definable (RadioRA 2 only).
```

Reply:
Yes, that sounds better.  I'll change it.